### DISCUSSION TITLE
Icon Request: fa-helm (Old timey ship helm/ship's wheel)

### DISCUSSION THREAD

**Discussion by miikkakostian** [FortAwesome/Font-Awesome#21223] [Icon Requests]
For all the helming needs.

**Comment by jinglesthula** [comment]
+1

**Comment by frankdl98** [comment]
+1

**Comment by xmuskrat** [comment]
+1

**Comment by xmuskrat** [comment]
Examples:

![image](https://user-images.githubusercontent.com/59917/61471333-71db3e80-a937-11e9-9bbb-542afc4ba0e5.png)

![image](https://user-images.githubusercontent.com/59917/61471356-79024c80-a937-11e9-810b-c0493c32edb5.png)

**Comment by rychoo2** [comment]
similar available in font awesome 5:
https://fontawesome.com/icons/dharmachakra?style=solid

**Comment by xmuskrat** [comment]
Kinda, but I know I'd be careful about misusing religious symbols, could offend people.

**Comment by jinglesthula** [comment]
I like the 6-spoke design, especially for smaller font sizes.  This would also avoid confusion with the dharmachakra icon referenced above.

**Comment by wiliamjk** [comment]
+1

**Comment by tcudjoe** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by Knud13** [comment]
+1 
(sorry if this isn't the correct way to vote up promote icon request I don't see any other way)

**Comment by tagliala** [comment]
@Knud13 by `+1` we mean an "upvote reaction"

<img width="334" height="167" alt="Image" src="https://github.com/user-attachments/assets/09ac19cd-4321-4ff9-a675-ff23df3fad98" />
